# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1235446811 18000
#      Mon Feb 23 22:40:11 2009 -0500
# Node ID 7557cf34ffcdac64d8e25ad2e3cae59d96f3fdf2
# Parent  4629e3925e75e098adfe28735e67540cf818aef6
ov-base-sparse.cc (octave_base_sparse<T>::print_raw): remove extra ")\n" from output

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,10 +1,13 @@
 2009-02-23  John W. Eaton  <jwe@octave.org>
 
+	* ov-base-sparse.cc (octave_base_sparse<T>::print_raw):
+	Remove extra ")" from output.
+
 	* DLD-FUNCTIONS/eigs.cc (Feigs): If sigma argument is not a
 	string, try extraction as complex value and check for error
 	instead of inquiring about type first.
 
 	* pt-eval.cc (tree_evaluator::visit_break_command,
 	tree_evaluator::visit_return_command,
 	tree_evaluator::visit_global_command,
 	tree_evaluator::visit_static_command): Handle breakpoint.
diff --git a/src/ov-base-sparse.cc b/src/ov-base-sparse.cc
--- a/src/ov-base-sparse.cc
+++ b/src/ov-base-sparse.cc
@@ -281,17 +281,17 @@ octave_base_sparse<T>::print_raw (std::o
 
   os << "Compressed Column Sparse (rows = " << nr
      << ", cols = " << nc
      << ", nnz = " << nz;
 
   double dnel = matrix.numel ();
 
   if (dnel > 0)
-    os << " [" << std::setprecision (2) << (nz / dnel * 100) << "%])\n";
+    os << " [" << std::setprecision (2) << (nz / dnel * 100) << "%]";
 
   os << ")\n";
 
   // add one to the printed indices to go from
   //  zero-based to one-based arrays
 
   if (nz != 0)
     {
